packages/trpc/server/routers/publicViewer/_router.tsx

```diff
@@ -22,6 +24,14 @@ export const publicViewerRouter = router({
     const handler = await importHandler(namespaced("countryCode"), () => import("./countryCode.handler"));
     return handler(opts);
   }),
+  submitRating: publicProcedure.input(ZSubmitRatingInputSchema).mutation(async (opts) => {
+    const handler = await importHandler(namespaced("submitRating"), () => import("./submitRating.handler"));
+    return handler(opts);
+  }),
+  noShow: publicProcedure.input(ZNoShowInputSchema).mutation(async (opts) => {
+    const handler = await importHandler(namespaced("noShow"), () => import("./noShow.handler"));
+    return handler(opts);
+  }),
```

Comment:
For a follow up. Generate a token from the email and verify it to avid exposing mutations publicly. Similarly on how we do for confirmation emails:

https://github.com/calcom/cal.com/blob/386968f216a5b868a8a4dac9f47e20fb5810af25/packages/emails/src/templates/OrganizerRequestEmail.tsx#L8-L11